src/hotspot/share/runtime/continuation.cpp

```diff
@@ -160,6 +160,13 @@ freeze_result Continuation::try_preempt(JavaThread* target, oop continuation) {
   freeze_result res = CAST_TO_FN_PTR(FreezeContFnT, freeze_preempt_entry())(target, target->last_Java_sp());
   log_trace(continuations, preempt)("try_preempt: %d", res);
   JVMTI_ONLY(jubm.set_result(res);)
+
+  if (target->has_pending_exception()) {
```

Comment:
Is `target` always the current thread, because we should only ever check and clear exceptions on the current thread. ??

Reply:
Yes. For context, this method was copied from the external preemption work which implemented the `Continuation.tryPreempt` API, where a thread tries to preempt another one. But that was all experimental and currently target is always current. I could rename the instances to avoid confusion.

Reply:
Yes please!

Reply:
Fixed.